### PR TITLE
save/load reorder columns

### LINKED ISSUE #2889: Cookie feature (re-order, resize)
Add feature to Cookie extension in order to save the re-order and resize configuration

### PR DESCRIPTION
**Pull request by UtechtDustin**
fix #2889 

Cant provide a example, because its not working on the live editor (i dont know why).
So please check it on your locale pc.

Peek 2020-02-02 02-29

### CODE REVIEW

File: src/extensions/reorder-columns/bootstrap-table-reorder-columns.js [hunk 1]
```diff
@@ -121,8 +121,16 @@ $.BootstrapTable = class extends $.BootstrapTable {
...
-      beforeStop: () => {
+      restoreState: order ? order : this.columnsSortOrder,
+      beforeStop: (table) => {
+        const sortOrder = {}
+        table.el.find('th').each((i, el) => {
+          sortOrder[$(el).data('field')] = i
+        })
+
+        this.columnsSortOrder = sortOrder
+        this.persistReorderColumnsState(this)
```
Comment (djhvscf) [inline src/extensions/reorder-columns/bootstrap-table-reorder-columns.js]: What happen if cookie is not used? Throw an exception?

Comment (UtechtDustin) [reply src/extensions/reorder-columns/bootstrap-table-reorder-columns.js]: Nothing happens :-)

Comment (djhvscf) [reply src/extensions/reorder-columns/bootstrap-table-reorder-columns.js]: ok, cool

File: src/extensions/resizable/bootstrap-table-resizable.js [hunk 2]
```diff
@@ -8,7 +8,9 @@ const isInit = that => that.$el.data('resizableColumns') !== undefined
 
 const initResizable = that => {
   if (that.options.resizable && !that.options.cardView && !isInit(that)) {
-    that.$el.resizableColumns()
+    that.$el.resizableColumns({
+      store: window.store
```
Comment (djhvscf) [inline src/extensions/resizable/bootstrap-table-resizable.js]: Why this change here?

Comment (UtechtDustin) [reply src/extensions/resizable/bootstrap-table-resizable.js]: Its to save the column resizing, which is already implemented in the [jquery-resizable](https://github.com/dobtco/jquery-resizable-columns) lib.

Comment (djhvscf) [reply src/extensions/resizable/bootstrap-table-resizable.js]: Ok, thanks!
